src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -2070,8 +2059,6 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<System
             void System.IDisposable.Dispose() { }
         }
     }
-
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
This is still needed. It's a bummer but unfortunately we don't have a way to avoid this removal as I explained offline.